examples/response-metrics/response_metrics.conf

```diff
@@ -0,0 +1,24 @@
+location = /report_metric {
+  internal;
+  resolver 8.8.8.8;
```

Comment:
Ideally, we would not have to define a resolver here, but unlike the other conf blocks, the default resolver doesn't appear to be picked up here.